src/test/unit_tests/test_ns_list_objects.js

```diff
@@ -313,6 +313,32 @@ function test_ns_list_objects(ns, object_sdk, bucket) {
         });
     });
 
+    mocha.describe('list objects - hidden dir', function() {
```

Comment:
how this test is relevant to the added fix? per my understanding this test is not creating a hidden directory that starts with .noobaa-nsfs except the one that is getting created during the object upload it won't list it as well